packages/addons/vitest-addon/index.ts

```diff
@@ -38,63 +38,143 @@ export default defineAddon({
 			`;
 		});
 
-		sv.file(`vite.config.${ext}`, (content) => {
-			const { ast, generateCode } = parseScript(content);
-
-			// find `defineConfig` import declaration for "vite"
-			const importDecls = ast.body.filter((n) => n.type === 'ImportDeclaration');
-			const defineConfigImportDecl = importDecls.find(
-				(importDecl) =>
-					(importDecl.source.value === 'vite' || importDecl.source.value === 'vitest/config') &&
-					importDecl.importKind === 'value' &&
-					importDecl.specifiers?.some(
-						(specifier) =>
-							specifier.type === 'ImportSpecifier' && specifier.imported.name === 'defineConfig'
-					)
-			);
-
-			// we'll need to replace the "vite" import for a "vitest/config" import.
-			// if `defineConfig` is the only specifier in that "vite" import, remove the entire import declaration
-			if (defineConfigImportDecl?.specifiers?.length === 1) {
-				const idxToRemove = ast.body.indexOf(defineConfigImportDecl);
-				ast.body.splice(idxToRemove, 1);
-			} else {
-				// otherwise, just remove the `defineConfig` specifier
-				const idxToRemove = defineConfigImportDecl?.specifiers?.findIndex(
-					(s) => s.type === 'ImportSpecifier' && s.imported.name === 'defineConfig'
-				);
-				if (idxToRemove) defineConfigImportDecl?.specifiers?.splice(idxToRemove, 1);
-			}
+		if (kit) {
```

Comment:
why only with kit? you can use component testing where you'd need testing library outside of kit too